plexapi/collection.py

```diff
@@ -194,6 +194,17 @@ def items(self):
             self._items = items
         return self._items
 
+    def visibility(self):
+        """ Returns the :class:`~plexapi.library.HubManage` for this collection. """
+        key = '/hubs/sections/%s/manage?metadataItemId=%s' % (self.section().key, self.ratingKey)
+        data = self._server.query(key)
+        try:
+            return self.findItems(data, cls=HubManage)[0]
+        except IndexError:
+            hub = HubManage(self._server, data, parent=self)
+            hub._collection_key = self.ratingKey
+            return hub
```

Comment:
We can set the `identifier`, `title`, and `_promoted` flag for an unmanaged collection.
```suggestion
        key = f'/hubs/sections/{self.librarySectionID}/manage?metadataItemId={self.ratingKey}'
        data = self._server.query(key)
        hub = self.findItem(data, cls=ManagedHub)
        if hub is None:
            hub = ManagedHub(self._server, data, parent=self)
            hub.identifier = f'custom.collection.{self.librarySectionID}.{self.ratingKey}'
            hub.title = self.title
            hub._promoted = False
        return hub
```